src/main/java/UITestFramework/CreateSession.java

```diff
@@ -222,19 +220,19 @@ public void startAppiumServer(String os) throws ExecuteException, IOException, I
 
 		}
 		else if (os.contains("mac os x")){
-			CommandLine command = new CommandLine("/Applications/Appium.app/Contents/Resources/node/bin/node");  
-			command.addArgument("/Applications/Appium.app/Contents/Resources/node_modules/appium/bin/appium.js", false);  
-			command.addArgument("--address", false);  
-			command.addArgument("127.0.0.1");  
-			command.addArgument("--port", false);  
-			command.addArgument("4723");  
-			command.addArgument("--full-reset", false);  
-			DefaultExecuteResultHandler resultHandler = new DefaultExecuteResultHandler();  
-			DefaultExecutor executor = new DefaultExecutor();  
-			executor.setExitValue(1);  
-			executor.execute(command, resultHandler);  
-			Thread.sleep(5000);  
+			String appiumJSPath = "/Users/mac/.npm/lib/node_modules/appium/build/lib/main.js";
+			builder = new AppiumServiceBuilder()
+					.withAppiumJS(new File(appiumJSPath))
+//                .usingDriverExecutable(new File(Appium_Node_Path))
+					.withIPAddress("0.0.0.0")
```

Comment:
`.withIPAddress("0.0.0.0")` is not required. It is default IP address

Reply:
This line is also not required.